elements/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -394,16 +394,18 @@ export class MultisignatureTransaction extends BaseTransaction {
 		if (!raw.m_keysgroup) {
 			return undefined;
 		}
+
+		// When syncing, nodes should receive `m_keysgroup` as csv string and then split the values into an array
+		// Due to the issue https://github.com/LiskHQ/lisk-sdk/issues/3612, v1.6 nodes will send `m_keysgroup` as an array thus skipping the array convertion
 		const multisignature = {
 			min: raw.m_min,
 			lifetime: raw.m_lifetime,
-			keysgroup: [],
+			keysgroup:
+				typeof raw.m_keysgroup === 'string'
+					? raw.m_keysgroup.split(',')
+					: raw.m_keysgroup,
```

Comment:
Will it be fine when `raw.m_keysgroup` is `null`?

Reply:
Yes, because of line `394`

Reply:
I've just pushed a new test case scenario covering this ;)